src/SFML/Audio/SoundFileWriterWav.cpp

```diff
@@ -27,8 +27,8 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/Audio/SoundFileWriterWav.hpp>
 #include <SFML/System/Err.hpp>
+#include <SFML/System/Utils.hpp>
 #include <algorithm>
```

Comment:
Can you check if this include can be removed now? `<algorithm>` is quite expensive as a header, would be nice to get rid of it as part of this PR if it's unnecessary.

Reply:
Looks like it can. I'll remove it.